packages/fds-components/src/components/interactive/Toast.jsx

```diff
@@ -0,0 +1,139 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+import Button from '@material-ui/core/Button';
+import IconButton from '@material-ui/core/IconButton';
+import FDS from 'fds-dictionary/dist/js/styleConstants';
+import CheckIcon from 'fds-icons/lib/react/CheckIcon';
+import InformationIcon from 'fds-icons/lib/react/InformationIcon';
+import DenyIcon from 'fds-icons/lib/react/DenyIcon';
+import Flex from '../layout/Flex';
+import FlexItem from '../layout/FlexItem';
+import CountdownButton from '../media/CountdownButton';
+import customPropTypes from '../../util/customPropTypes';
+
+const Toast = ({
+  content,
+  type,
+  actionLabel,
+  progress,
+  onAction,
+  isAutoDismiss,
+  dismissDelay,
+  canDismiss,
+  dismissToast /* Passed from Toaster */,
+}) => {
+  const classNames = cx('toast', `toast--${type}`);
+
+  const icon = {
+    info: <InformationIcon size="l" color={FDS.COLOR_AQUA} />,
+    success: <CheckIcon size="xs" color={FDS.COLOR_WHITE} />,
+    warn: (
+      <div role="image" className="typemod--white">
+        !
+      </div>
+    ),
+    error: (
+      <div role="image" className="typemod--white">
+        !
+      </div>
+    ),
+  }[type];
+
+  const useContinuousProgressBar = type === 'progress' && progress;
+  const useAmountProgressBar = type === 'progress' && !progress;
+
+  const toastBar = (
+    <div className="toast-bar">
+      {useContinuousProgressBar && (
+        <div
+          className="toast-progressBar toast-progressBar--amount"
```

Comment:
looks like `toast-progressBar--amount` should be swapped with `toast-progressBar--continuous`